utopia-remix/app/styles/button.css.ts

```diff
@@ -1,28 +1,28 @@
-import { recipe } from '@vanilla-extract/recipes'
-import { sprinkles } from './sprinkles.css'
+import { recipe } from "@vanilla-extract/recipes";
```

Comment:
hmmmmmm it looks like Prettier did not kick in correctly here 🧐 (or even more plausible, this is a remnant from the pre-prettier-fix (https://github.com/concrete-utopia/utopia/pull/4857) era